a3p-integration/README.md

```diff
@@ -1,38 +1,95 @@
 
-For each proposal, their package.json is also separate and can't access the SDK code. There is an issue to automatically build proposals from scripts declared in the proposal package.json: https://github.com/Agoric/agoric-3-proposals/issues/87 . Until that is resolved, use `agoric run` on the proposal and copy the outputs to a `submission` directory within the proposals package, to be checked in.
+## Hooks
+
+Each proposal must have a `test.sh` file, which is executed during a `synthetic-chain test` invocation. In general this simply executes `yarn ava` to run any `*.test.js`.
+
+A proposal can also have a `use.sh` file to perform any persisted action after the proposal has passed. The effects of these actions can be used in the test phase, or in subsequent proposal layers.
+
+A chain software upgrade proposal can also have a `prepare.sh` file which is executed before the proposal is submitted to the chain. The actions taken are similarly persisted.
+
+Instead of relying on an automatic `submission` folder, a core-eval proposal can define an `eval.sh` file to generate and submit the core eval (or perform any standalone actions).
 
 # Troubleshooting
 
 ## no match for platform
 
 If you get an error like this,
 ```
-ERROR: failed to solve: ghcr.io/agoric/agoric-3-proposals:main: no match for platform in manifest sha256:83321abda66fa94915f1ae20d651b66870f2d1aac17b71449c04ecd46b6b1b96: not found
+ERROR: failed to solve: ghcr.io/agoric/agoric-3-proposals:latest: no match for platform in manifest sha256:14e22b6f75b568a5d32f7a74b701d978b7656ba4d33e2ec7ad2ff0611e7c2530: not found
 ```
-it's because our CI only builds x64 yet and you're on some other machine, probably a Mac.
+it's because you're on an architecture for which we don't generate an image. We currently generate `linux/amd64` and `linux/arm64` images.
 
-There is some effort to make CI build multiplatform: https://github.com/Agoric/agoric-3-proposals/pull/32
-
-Meanwhile you can build the `main` image locally:
+You can build the `latest` image locally:
 
 ```sh
 cd agoric-3-proposals
-./node_modules/.bin/synthetic-chain build
+yarn install
+yarn synthetic-chain build
 
 # build the default entrypoint and tag it so the `append` command finds it
-docker buildx build --tag ghcr.io/agoric/agoric-3-proposals:main .
+docker buildx build --tag ghcr.io/agoric/agoric-3-proposals:latest .
```

Comment:
thanks, I missed updating docs in https://github.com/Agoric/agoric-sdk/pull/8873  (which affirms the stale docs risk)